.github/scripts/smoke_test.sh

```diff
@@ -0,0 +1,78 @@
+#! /bin/bash
+
+# Software test
+test_software() {
+cd ../../.. && \
+if [ -d "./$project" ]; then
+    rm -r $project
+fi
+dbt init -s $project && \
+{ cat > ./.dbt/profiles.yml << EOF
+$project:
+  outputs:
+    dev:
+      password: $test_password
+      port: 9047
+      software_host: $test_host
+      threads: 1
+      type: dremio
+      use_ssl: false
+      user: $test_user
+  target: dev
+EOF
+} && \
+
+cd $project/ && \
+dbt debug && \
+dbt run
+
+}
+
+#Cloud test
+test_cloud() {
+cd ../../.. && \
+if [ -d "./$project" ]; then
+    rm -r $project
+fi
+dbt init -s $project && \
+{ cat > ./.dbt/profiles.yml << EOF
+$project:
+  outputs:
+    dev:
+      cloud_host: $test_host
+      cloud_project_id: $test_cloud_project
+      pat: $test_token
+      threads: 1
+      type: dremio
+      use_ssl: true
+      user: $test_user
+  target: dev
+EOF
+} && \
+echo 'vars:' >> ./$project/dbt_project.yml && \
+echo '  dremio:reflections_enabled: false' >> ./$project/dbt_project.yml && \
+
+cd $project/ && \
+dbt debug && \
+dbt run 
+}
+
+# Main
+test_type=$1
+
+if [ $test_type == software ]; then 
```

Comment:
We're missing testing PAT for software.

Reply:
I purposely left it out, since I thought one test for software and one for cloud were adequate for smoke tests. BUT I'm not against adding PAT for software.

Reply:
I think it'd be nice to have 3 test types: softwareUP, softwarePAT and cloud. I feel as though PAT is something we should test against as well.